weaver/core/drivers/corda-driver/README.md

```diff
@@ -120,7 +120,7 @@ To push image to github container registry:
     - `DRIVER_RPC_PASSWORD`: Password for the above RPC user.
     - `EXTERNAL_NETWORK`: Name of the docker network in which the Corda containers are deployed.
     - `DOCKER_IMAGE_NAME`: _Keep this unchanged_.
-    - `DOCKER_TAG`: Refer here for the image tags available: [weaver-corda-driver](https://github.com/hyperledger-labs/weaver-dlt-interoperability/pkgs/container/weaver-corda-driver)
+    - `DOCKER_TAG`: Refer here for the image tags available: [cacti-weaver-corda-driver](https://github.com/hyperledger/cacti/pkgs/container/cacti-weaver-corda-driver)
```

Comment:
If we change the package name to `cacti-weaver-driver-corda`, make sure to edit this too.